TESTING.md

```diff
@@ -98,6 +98,17 @@ See [anomaly-detection#158](https://github.com/opensearch-project/anomaly-detect
 
 Each plugin can test various functionalities in their bwc tests and add more scenarios if needed.
 
+#### BWC tests on distribution bundle level
```

Comment:
TOC needs updating, please?

Reply:
@dblock Thanks dB. Just updated the table. Thanks!